cypress/integration/articleFlows/commentOnArticle.spec.js

```diff
@@ -208,27 +208,32 @@ describe('Comment on articles', () => {
         { fixture: 'search/usernames.json' },
       );
 
-      cy.findByLabelText(/^Add a comment to the discussion$/i).click();
       // Get a handle to the newly substituted textbox
-      cy.findByRole('textbox', {
-        name: /^Add a comment to the discussion$/i,
-      }).type('first comment');
+      cy.findByRole('main')
+        .as('main')
+        .findByRole('textbox', /^Add a comment to the discussion$/i)
+        .focus()
```

Comment:
This appears to be the piece causing the flakiness. Switching from a `.click()` to `.focus()` seems to do the trick.